rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -952,6 +954,27 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
     errorLog.readAndClearErrorVector should be(Vector.empty[InterpreterError])
   }
 
+  "eval of a method" should "substitute target before evaluating" in {
+    implicit val errorLog = new ErrorLog()
+    val costAccounting =
+      CostAccountingAlg.monadState(AtomicRefMonadState.of[Task, CostAccount](CostAccount.zero))
+    val splitRand = rand.splitByte(0)
+    val nthCall: Expr =
```

Comment:
copy-paste error in name :)

Reply:
Next time s/b adds a test for a method call we should probably extract a test util here